storageapi/src/vespa/storageapi/messageapi/storagemessage.h

```diff
@@ -263,40 +263,44 @@ class MessageType : public vespalib::Printable {
  */
 class StorageMessageAddress {
 public:
-    enum Protocol { STORAGE, DOCUMENT };
+    enum class Protocol : uint8_t { STORAGE, DOCUMENT };
 
 private:
-    vespalib::string  _cluster;
+    const vespalib::string  *_cluster;
     // Used for internal VDS addresses only
-    size_t               _precomputed_storage_hash;
-    const lib::NodeType* _type;
-    Protocol             _protocol;
-    uint16_t             _index;
+    uint32_t                 _precomputed_storage_hash;
+    lib::NodeType::Type      _type;
+    Protocol                 _protocol;
+    uint16_t                 _index;
 
 public:
     StorageMessageAddress(); // Only to be used when transient default ctor semantics are needed by containers
-    StorageMessageAddress(vespalib::stringref clusterName,
-                          const lib::NodeType& type, uint16_t index,
-                          Protocol protocol = STORAGE);
+    StorageMessageAddress(const vespalib::string * clusterName, const lib::NodeType& type, uint16_t index);
```

Comment:
_clusterName_ -> _cluster_ (as elsewhere in the file).

Reply:
Fixed